src/app/item-popup/PlugTooltip.tsx

```diff
@@ -25,14 +31,10 @@ export default function PlugTooltip({
 }) {
   // TODO: show insertion costs
   const defs = useD2Definitions();
```

Comment:
Are we loading refs here just to pass it down?

Reply:
Maybe, it was pretty late when I did this. I will have another look over it and push it down if it makes sense.